hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/virtual/ViewTable.java

```diff
@@ -41,8 +41,8 @@ public class ViewTable extends Table {
     private final String viewQuery;
     private RelNode viewRel;
 
-    public ViewTable(String schemaName, String viewName, String viewQuery, TableStatistics statistics) {
-        super(schemaName, viewName, null, statistics);
+    public ViewTable(String schemaName, String viewName, String viewQuery, TableStatistics statistics, boolean streaming) {
+        super(schemaName, viewName, null, statistics, "View", streaming);
```

Comment:
Overall, I am coming to the conclusion we can avoid defining 'streaming' view here and do it later based on `OptUtils.isUnbounded(viewRel)` in `initFields` method. WDYT?

Reply:
So basically always pass e.g. `false` and set "real" value in initFields? I'm not that familiar with SQL yet, but if initFields is mandatory, then it makes sense

Reply:
bf9c2cc6ec4e4be6414e049dac0cc5ef419a8a5a